### PR TITLE
Dev daniel

### PR DESCRIPTION
**Pull request by danielbattistini**
Add Lc syst

### CODE REVIEW

**Review by fgrosa — APPROVED** [review]
It looks ok to me!

File: GetRawYieldsDplusDs.C [hunk 1]
```diff
@@ -535,6 +535,10 @@ int GetRawYieldsDplusDs(int cent, bool isMC, TString infilename, TString cfgfile
                 massFitter->SetFixRatio2GausSigma(
                 hSigmaToFix1->GetBinContent(iPt+1)/hSigmaToFix2->GetBinContent(iPt+1));
             }
+            if(fixSigmaRatio){
+                massFitter->SetFixRatio2GausSigma(
+                hSigmaToFix1->GetBinContent(iPt+1)/hSigmaToFix2->GetBinContent(iPt+1));
+            }
```
Comment (fcatalan92) [inline GetRawYieldsDplusDs.C:541]: Actually, this is a duplicate of Lines 534-537

Comment (fgrosa) [reply GetRawYieldsDplusDs.C:541]: indeed, I did not notice

Comment (fcatalan92) [reply GetRawYieldsDplusDs.C:541]: Fixed by #247

**Pull request merged**